enterprise-search/examples/question_answering.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>Change EnterpriseSearchRetriever and langchain types to use Code Font</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>